server/app/controllers/meetingfreqs_controller.rb

```diff
@@ -0,0 +1,34 @@
+class MeetingfreqsController < ApplicationController
```

Comment:
I would recommend not to shorten words. It's just a place to create misunderstandings.